setup.cfg

```diff
@@ -33,5 +33,4 @@ markers =
 known_localfolder = nemo,tests
 sections = FUTURE,STDLIB,THIRDPARTY,LOCALFOLDER
 default_section = THIRDPARTY
-#TODO tests/unit/core/test_deploy_export.py gets screwed by isort
-skip = setup.py, nemo/utils/__init__.py, tests/unit/core/test_deploy_export.py
```

Comment:
please put this back - there are some issues around black/isort on some of dev machines which required us to add skip test_deploy_export

Reply:
ok